torch/_torch_docs.py

```diff
@@ -8930,6 +8930,54 @@ def merge_dicts(*dicts):
 """.format(**factory_common_args))
 
 
+add_docstr(torch.kaiser_window, """
+kaiser_window(window_length, beta, periodic=True, *, dtype=None, \
+layout=torch.strided, device=None, requires_grad=False) -> Tensor
+""" + r"""
+Kaiser window function.
+
+.. math::
+    w[n] = I_0 \left( \beta \sqrt{1 - \frac{4n^2}{(M-1)^2}} \right) / I_0( \beta )
+
+with
+
+.. math::
+    -\frac{M-1}{2} \leq n \leq \frac{M-1}{2}
+
+where :math:`M` is the full window size.
+
+If :attr:`periodic` is true, the :math:`M` in
```

Comment:
This is a tricky paragraph. Let me suggest another idea. If `M` is defined in advance we can say when it's defined:

"Letting i0 be the zeroth order modified Bessel function of the first kind (see torch.i0) and M = :attr:\'window_length\' if :attr:\'periodic\' is False and :attr:\`window_length\` + 1 if :attr:\`periodic\` is True, this computes:"

Then the mathematical expression as dicussed above. 

Then after the mathematical expression we have this paragraph, which can elaborate on the intent of the periodic argument. Maybe something like:

"Calling torch.kaiser_window(L, B, periodic=True) is equivalent to torch.kaiser_window(L + 1, B, periodic=False)[:-1]). The :attr:\`periodic\` argument is intended as a helpful shorthand to produce a periodic window as input to functions like :func:\`torch.stft\`."

Note the use of :func: and not :meth for the torch.stft reference. 

Does that split sound more reasonable?